src/observation_recipe.jl

```diff
@@ -33,6 +34,64 @@ and return a noise covariance matrix.
 """
 abstract type AbstractCovarianceEstimator end
 
+"""
+    ScalarCovariance <: AbstractCovarianceEstimator
+
+Contain the necessary information to construct the scalar covariance matrix.
+"""
+struct ScalarCovariance{FT1 <: AbstractFloat, FT2 <: AbstractFloat} <:
+       AbstractCovarianceEstimator
+    """Scalar to multiply the identity matrix by"""
+    scalar::FT1
+
+    """Use latitude weights"""
+    use_latitude_weights::Bool
+
+    """The minimum cosine weight when using latitude weighting"""
+    min_cosd_lat::FT2
+end
+
+"""
+    ScalarCovariance(;
+        use_latitude_weights = false,
+        min_cosd_lat = 0.1,
+    )
+
+Create a `ScalarCovariance` which specifies how the covariance matrix
+should be formed. When used with `ObservationRecipe.observation` or
+`ObservationRecipe.covariance`, return a `Diagonal` matrix.
+
+Keyword arguments
+=====================
+
+- `scalar`: 
```

Comment:
This is missing a description
```suggestion
- `scalar`: Scalar value to multiply the identity matrix by.
```